.changeset/ten-buckets-exist.md

```diff
@@ -0,0 +1,41 @@
+---
+'xstate': patch
+---
+
+Add `assertEvent()` helper.
```

Comment:
I feel like we might want to add a note that this throws if the assertion fails. That's not entirely clear from the example alone imo, and that's all most users will see in the release notes.

Reply:
Roger, will update the changeset

Reply:
Expanded the changeset docs